lib/rules/validate-jsdoc/enforce-existence.js

```diff
@@ -80,6 +90,18 @@ function enforceExistence(node, err) {
             return;
         }
     }
+    if (policy.noParamsAndReturns === false) {
+        var numberOfReturnsWhichRequireJsdoc = 0;
+        this._iterate(function(n) {
+            if (n && n.type === 'ReturnStatement' && n.argument) {
+                numberOfReturnsWhichRequireJsdoc++;
```

Comment:
Does this looking for `return`s inside nested functions?

Looks like here's a bug. Can you add a test for:
```
function () {
  function () {
    return 1;
  ]
}
```